todo/tests/integration/base_mongo_test.py

```
@@ -0,0 +1,27 @@
+        cls.override = override_settings(
+            MONGODB_URI=cls.mongo_url,
+            DB_NAME="testdb",
+        )
+        cls.override.enable()
+        DatabaseManager().reset()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.mongo_client.close()
+        cls.override.disable()
+        super().tearDownClass()
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)